eva/executor/executor_utils.py

```diff
@@ -91,11 +92,17 @@ def validate_video(video_path: Path) -> bool:
         )
 
 
+def validate_document(doc_path: Path) -> bool:
+    return doc_path.suffix in LOADER_MAPPING
+
+
 def validate_media(file_path: Path, media_type: FileFormatType) -> bool:
     if media_type == FileFormatType.VIDEO:
         return validate_video(file_path)
     elif media_type == FileFormatType.IMAGE:
         return validate_image(file_path)
+    elif media_type == FileFormatType.DOCUMENT:
```

Comment:
I saw langchain support many different types of documents. Are we going to validate them all? And what is the standard way of validating those documents?

Reply:
I don't plan to validate all documents. Just the ones we support in our minimal version

Reply:
We can use -- https://pypi.org/project/python-magic/ -- in a future PR to check file types beyond just suffix matching for security